src/api/plugins/tdbp/warren_tdbp/api.py

```diff
@@ -0,0 +1,169 @@
+"""Warren API v1 tdbp router."""
+
+import logging
+from datetime import date, datetime
+from typing import Optional
+
+from fastapi import APIRouter, HTTPException, status
+from warren.exceptions import LrsClientException
+
+from .indicators import Cohort, Grades, Scores, SlidingWindow
+
+router = APIRouter(
+    prefix="/tdbp",
+)
+
+logger = logging.getLogger(__name__)
+
+
+@router.get("/window")
+async def get_sliding_window(
+    course_id: str,
+    until: Optional[date],
+):
+    """Return course sliding window indicator.
+
+    Args:
+        course_id (str): The course identifier on Moodle.
+        until (date): End date until where to compute the sliding window.
```

Comment:
```suggestion
        until (date): End date until when to compute the sliding window.
```
same for the next `until`s